pkg/kf/commands/apps/apps.go

```diff
@@ -37,47 +40,100 @@ func NewAppsCommand(p *config.KfParams, appsClient apps.Client) *cobra.Command {
 			}
 			cmd.SilenceUsage = true
 
-			fmt.Fprintf(cmd.OutOrStdout(), "Getting apps in namespace: %s\n", p.Namespace)
+			fmt.Fprintf(cmd.OutOrStdout(), "Getting apps in space %s\n", p.Namespace)
 
 			apps, err := appsClient.List(p.Namespace)
 			if err != nil {
 				return err
 			}
-			fmt.Fprintf(cmd.OutOrStdout(), "Found %d apps in namespace %s\n", len(apps), p.Namespace)
 			fmt.Fprintln(cmd.OutOrStdout())
 
-			// Emulating:
-			// https://github.com/knative/serving/blob/master/config/300-service.yaml
 			w := tabwriter.NewWriter(cmd.OutOrStdout(), 8, 4, 1, ' ', tabwriter.StripEscape)
-			fmt.Fprintln(w, "NAME\tDOMAIN\tLATEST CREATED\tLATEST READY\tREADY\tREASON")
+			fmt.Fprintln(w, "name\t\trequested state\t\tinstances\tmemory\tdisk\turls")
```

Comment:
There are double `\t`s between name/requested state and requested state/instances

Reply:
I actually did that for readability, but I'm not attached to it. WDYT?